plugins/admission/pkg/pluto/pluto.go

```diff
@@ -42,9 +63,54 @@ func ProcessPluto(input []byte) (models.ReportInfo, error) {
 	if err != nil {
 		return report, err
 	}
+	_ = apiInstance.DisplayOutput()
```

Comment:
why do we need this?

Reply:
Ooo - thank you, that was left over troubleshooting to display the Pluto struct in admission logs. Removed.